media_kit_video/lib/src/video_controller_android.dart

```diff
@@ -144,27 +214,10 @@ class VideoControllerAndroid extends VideoController {
   Future<void> setSize({
     int? width,
     int? height,
-  }) async {
-    final handle = await player.handle;
-    if (this.width == width && this.height == height) {
-      // No need to resize if the requested size is same as the current size.
-      return;
-    }
-    this.width = width;
-    this.height = height;
-    // ----------------------------------------------
-    NativeLibrary.ensureInitialized();
-    final mpv = MPV(DynamicLibrary.open(NativeLibrary.path));
-    final name = 'android-surface-size'.toNativeUtf8();
-    final value = '${width}x$height'.toNativeUtf8();
-    mpv.mpv_set_option_string(
-      Pointer.fromAddress(handle),
-      name.cast(),
-      value.cast(),
+  }) {
+    throw UnimplementedError(
```

Comment:
I think it should be added to the changelog that a throw has been added to `setSize`, otherwise some people will be surprised ^^

Reply:
Sure!